src/components/RewardCard.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import ReactCardFlip from 'react-card-flip'
+
+const Container = styled.div`
+  background-image: ${p => p.theme.colors.backgroundTertiary};
+  border-radius: 8px;
+  padding: 20px;
+  width: 280px;
+  height: 160px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+`
+
+const Title = styled.h1`
+  font-size: 24px;
+  font-weight: 400;
+  color: ${p => p.theme.colors.lines};
+`
+
+const Text = styled.h3`
+  font-size: 18px;
+  font-weight: 300;
+  margin-top: -12px;
+  color: ${p => p.theme.colors.lines};
+`
+
+const Description = styled.h3`
+  font-size: 18px;
+  font-weight: 300;
+  text-align: center;
+  color: ${p => p.theme.colors.lines};
+`
+
+const SubTitle = styled.p`
+  font-size: 14px;
+  font-weight: 700;
+  color: ${p => p.theme.colors.bar};
+  text-align: right;
+  margin-top: 8px;
+`
```

Comment:
should these colours be `theme.colors.text`?

Reply:
oh idk why it didn't highlight correctly, but i was talking about all the text colours

Reply:
it should be the same color as the bar! it's for the points here above the bar ![image](https://github.com/user-attachments/assets/484321df-78b3-49c3-ba81-2b33d70e58cc)

Reply:
ohhh okay yeah that sounds good! should the the other ones should be `colors.text` instead of `colors.lines`?
![16065](https://github.com/user-attachments/assets/12d9f8e5-3cde-48c9-9bcd-b1e9a4986419)